### DISCUSSION TITLE
Is Github Copilot offered in Github students pack same as paid one?

### DISCUSSION THREAD

**Discussion by sPredictorX1708** [community/community#112975] [GitHub Education]
### Select Topic Area

Question

### Body

I was wondering if github students pack copilot is same as the paid one? Are they using same language model?

**Comment by EliasJHL** [comment]
Hello,

Yes, they use the same language model, but the copilot education is a limited version of the paid one.

**Reply by sPredictorX1708** [reply]
By limited version what do you mean? Is it not same as Github Copilot Individual plan?

**Reply by sPredictorX1708** [reply]
It says - 

```
Free access to Copilot Individual while you're a student. To enable Copilot, go to your account settings and under Code, planning, and automation, select Copilot. You'll see "You're eligible to use copilot for free" and a button to claim the offer.
```

**Comment by EliasJHL** [comment]
Hello,

The only limitation I see with Copilot is in code review. Otherwise, the free and paid versions seem to offer similar functionality.
